features/storage/kvstore/tdbstore/TDBStore.cpp

```diff
@@ -24,7 +24,10 @@
 #include "mbed_error.h"
 #include "mbed_wait_api.h"
 #include "MbedCRC.h"
+//Bypass the check of NVStore co existance if compiled for TARGET_TFM
+#if !(TARGET_BYPASS_NVSTORE_CHECK)
```

Comment:
Why is this target macro not a config ? or rather is this true for any TFM, should there be more generic name for exclusion like this?

the commit 2acb1c5754abb9ee1817f61d1584973e78d43ce5 does not say much what is going on here.

Reply:
@yossi2le

Reply:
Also here BYPASS_NVSTORE_CHECK ?

Reply:
yes I know :( I am working on it. 1 minute and it will be fix

Reply:
Done